packages/wallet/src/services/AssetsTracker.ts

```diff
@@ -8,46 +8,31 @@ import { distinct, from, map, mergeMap, of, scan, tap } from 'rxjs';
 export const createAssetService =
   (assetProvider: TrackedAssetProvider, retryBackoffConfig: RetryBackoffConfig) => (assetId: Cardano.AssetId) =>
     coldObservableProvider(
-      () => assetProvider.getAsset(assetId),
+      () => assetProvider.getAsset(assetId, { history: true, nftMetadata: true, tokenMetadata: true }),
```

Comment:
Do we really need to fetch history here?

Reply:
I don't think so, I did it like this because the ticket said to "Update wallet package to fetch all extra data" but I might have misunderstood. I'll remove it